vignettes/quickstart_glmm.Rmd

```diff
@@ -0,0 +1,354 @@
+perform the projection for a submodel of desired size using the function
+`project`. The projection can also be coerced to a matrix with draws of the
+selected variables and sigma. The draws can be visualized with, for example, the
+`mcmc_areas` function in the `bayesplot` package. Below we compare how the
+projection affects the three most relevant variables.
+
+<!-- ```{r, fig.width=6, fig.height=2} -->
+<!--  # Visualise the three most relevant variables in the full model -\-> -->
+<!--  mcmc_areas(as.matrix(vs$refmodel$fit), -->
+<!--             pars = c("(Intercept)", "Subject__Intercept", "age", -->
+<!--                      "sd_Subject__age", "sigma")) -->
+<!-- ``` -->
+
+```{r, cache=TRUE, fig.width=6, fig.height=2, message=FALSE, warning=FALSE}
+# Visualise the projected three most relevant variables
+proj <- project(vs, nterms = 2, ns = 500)
+mcmc_areas(as.matrix(proj), pars = solution_terms(vs)[1:2])
+```
+
+We make predictions with the projected submodels. For point estimates we can use
+method `proj_linpred`. Test inputs can be provided using the keyword `newdata`.
+If also the test targets `ynew` are provided, then the function evaluates the
+log predictive density at these points. For instance, the following computes the
+mean of the predictive distribution and evaluates the log density at the
+training points using the 5 most relevant variables.
+
+```{r, cache=TRUE, message=FALSE, warning=FALSE}
+pred <- proj_linpred(vs, newdata = Orthodont, nterms = 5, integrated = TRUE)
+```
+
+Visualize the predictions
+```{r, fig.width=5, fig.height=3}
+ggplot() +
+  geom_point(aes(x = pred$pred, y = Orthodont$distance)) +
+  geom_abline(slope = 1, color = "red") +
+  labs(x = "prediction", y = "y")
+```
+
+We also obtain draws from the projected predictive distribution. Here's an
+example prediction for the first data point using 5 terms (the observed
+value is marked by the red line).
+
+```{r, fig.height=3, fig.width=5, message=FALSE, warning=FALSE}
+subset <- Orthodont %>% dplyr::sample_n(1)
+y_subset <- subset %>% dplyr::select(distance)
+y1_rep <- proj_predict(vs, newdata = subset, nterms = 5, seed = 7560)
+qplot(as.vector(y1_rep), bins = 25) +
+  geom_vline(xintercept = as.numeric(y_subset), color = "red") +
+  xlab("y1_rep")
+```
+
+## Poisson example
+
+In this case we will download count data from the web,
+
+```{r}
+data_pois <- read.table("data_pois.csv", header = TRUE)
+```
+
+These data correspond to a phylogenetic dataset where the phenotype is expressed
+as counts. This kind of data is relevant in evolutionary biology when data of
+many species are analyzed at the same time. The model we fit here is borrowed
+from *Modern Phylogenetic Comparative Methods and the application in
+Evolutionary Biology* (de Villemeruil & Nakagawa, 2014). The necessary data can
+also be downloaded from the corresponding website
+(http://www.mpcm-evolution.com/). The model is taken from the *Estimating
```

Comment:
Line 159 The model... 
Line 163 The model...
considering rewording one these sentences